src/transport/raw/MessageHeader.cpp

```diff
@@ -158,11 +160,22 @@ CHIP_ERROR PacketHeader::Decode(const uint8_t * const data, uint16_t size, uint1
     err = reader.Read32(&mMessageId).StatusCode();
     SuccessOrExit(err);
 
+#ifdef DEBUG
+    ChipLogDetail(Inet, "PacketHeader::Decode");
+    ChipLogDetail(Inet, "  header=%u", (int) header);
+    ChipLogDetail(Inet, "  mEncryptionType=%d", (int) mEncryptionType);
```

Comment:
Please use the right types in the print format instead of casting things to int.  Especially with C-style casts.

Reply:
addressed in latest